src/OpenSage.Game/Diagnostics/MainView.cs

```diff
@@ -140,8 +140,8 @@ public void Draw(ref bool isGameViewFocused)
                                 var playableSides = _context.Game.GetPlayableSides().ToList();
                                 //var faction1 = playableSides[random.Next(0, playableSides.Count())];
                                 //var faction2 = playableSides[random.Next(0, playableSides.Count())];
-                                var faction1 = playableSides[3];
-                                var faction2 = playableSides[3];
+                                var faction1 = playableSides[0];
```

Comment:
This still seems super weird. A faction picker would make more sense

Reply:
Go for it:)

Reply:
done